prow/tide/tide.go

```diff
@@ -246,11 +247,21 @@ func requirementDiff(pr *PullRequest, q *config.TideQuery) (string, int) {
 			desc = fmt.Sprintf(" Should not have %s labels.", strings.Join(trunced, ", "))
 		}
 	}
+	diff = len(missingLabels) + len(presentLabels)
+
+	if desc == "" {
+		for _, excludedBranch := range q.ExcludedBranches {
+			if string(pr.BaseRef.Name) == excludedBranch {
+				desc = fmt.Sprintf(" Merging to branch %s is forbidden.", pr.BaseRef.Name)
+			}
+		}
+		diff = 1
```

Comment:
We want to increase `diff` even if we don't end up showing a message about that requirement in the status description. See this review: https://github.com/kubernetes/test-infra/pull/7454#pullrequestreview-108208952